Src/BootCamp.Chapter/BalanceParser.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceParser
+    {
+        public static BalanceStats FindHighestBalance(string[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo("en-GB");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'en-GB' 4 times.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)